tests/e2e/configuration/library-mode/lightspeed-stack-rbac.yaml

```
@@ -99,3 +99,20 @@ authorization:
+      - rag_id: e2e-test-docs
+        backend: faiss
+        embedding_model: sentence-transformers/all-mpnet-base-v2
+        embedding_dimension: 768
+        vector_db_id: ${env.FAISS_VECTOR_STORE_ID}
+        db_path: ${env.KV_RAG_PATH:=~/.llama/storage/rag/kv_store.db}
+        score_multiplier: 1.0
+  retrieval:
+    tool:
+      sources:
+        - e2e-test-docs
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)